routes/orders/order.controller.js

```diff
@@ -13,6 +13,18 @@ function list(req, res, next) {
         .catch(e => next(e));
 }
 
+/**
+ * Return a list of incomplete orders.
+ * @param {*} req The express request object.
+ * @param {*} res The express result object.
+ * @param {*} next Next match route handler.
+ */
+function listInProgress(req, res, next) {
+    Order.get("InProgress")
+        .then(users => res.json(users))
```

Comment:
Users can be changed to data or items I think, something more specific to this context. I need to change it this in the counter functions as well...

Reply:
Ah yeah I didn't spot that - copy and paste overlook!